GuildedRoseLLCUITests/GuildedRoseLLCUITests.swift

```diff
@@ -10,7 +10,7 @@ class GuildedRoseLLCSpec: QuickSpec {
             app = XCUIApplication()
             app.launch()
         }
-        describe("the message") {
+        describe("the greeting message") {
```

Comment:
I would take a look at your test descriptions. "The greeting message for the greeting" feels a bit awkward to me. It's usually `describe("<thing>")`, `context("<situation / scenario>")`.

```swift
describe("The list of items") { // Alternatives: "The items page", "The welcome page"
  context("when there are no items") {
    it("shows a message saying there are no items') {
    } 
  }

  context("when there are items") {
  }
}
```